.github/actions/configure-credentials/action.yaml

```diff
@@ -8,13 +8,30 @@ inputs:
   aws_region:
     description: "The AWS region where the account resides"
     required: true
+  domain:
```

Comment:
We discussed this briefly - not sure if domain is legacy or still needed
If multi domain is passed it is hardcoded to use the management runner ; if is not can it not just do what it did before and use the runner derived from github.event.repository.name?